### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Manage Access Policies_Add_ScreenReader: NVDA does not read any status as row added when add and row removed when remove. 

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1981]
**“Check out Accessibility Insights! Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

**GitHubTags:** #A11y_AzureToolsStorageExplorer; #A11yWCAG2.1; #A11YTCS; #DesktopApp; #MAS4.1.3

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0

**Repro Steps:**
Launch Storage Explorer
Launch Storage Explorer
Navigate to "Open Connect dialog" pane and Sign in to Azure.
Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
Navigate to "Blob Containers" of the selected subscription, right click and Select "Create Blob Container" and hit enter.
Right Click on newly created blob container and select "Manage Access Policies" and hit enter.
Navigate to "Add" control and hit enter.
Start NVDA and using Tab key and check whether the Voiceover reads any status as row added when add and row removed when remove.

**​Actual:​**
NVDA does not read any status as row added when add and row removed when remove.

**Expected:**
NVDA should read the status as row added when add and row removed when remove.

**UserImpact**:
Screen reader users unable to know the which dialog opened if it will not read the title.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx 

**MAS Reference:
MAS4.1.3:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=13bc100c-7d1a-4d0c-baa4-2e137da441aa 

Attachment for reference
[1981_A11y_AzureToolsStorageExplorer_Manage Access Policies_Add_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3670131/1981_A11y_AzureToolsStorageExplorer_Manage.Access.Policies_Add_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
